src/index.css

```diff
@@ -14,12 +14,11 @@ body {
 :root {
   --overlay-color: rgba(0, 0, 0, 0.7);
   --primary-color: rgb(104, 255, 98);
-  --light-primary-color: #ffd35a;
-  --dark-primary-color: #cca027;
   --grey-color: #aaaaaa;
   --background-color: #1f1d2b;
   --dark-background-color: #1a1922;
   --transparent-color: rgba(0, 0, 0, 0.1);
```

Comment:
either use hex or rgba to be consistent!!

Reply:
Sure. Thanks for the suggestion